css/style.css

```diff
@@ -0,0 +1,206 @@
+body {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+}
+
+.flex {
+  display: flex;
+}
+
+.container {
+  width: 100%;
+  flex-direction: column;
+  align-items: center;
+}
+
+/* Main Section */
+
+.main-section {
+  flex-direction: column;
+  align-items: center;
+  width: 80%;
+  background-image: url(../images/main-section-background.jpg);
+  background-size: cover;
+  background-position-y: -750px;
+  height: 650px;
+}
+
+.navbar {
+  height: 36px;
+  margin-top: 50px;
+  width: 70%;
+}
+
+.navbar button {
+  color: white;
+  cursor: pointer;
+}
+
+.normal-box-1 {
+  background: linear-gradient(to top, #6e6e6e, #4d4d4d, #6b6b6b 55%, #6e6e6e 45%, #707070, #6e6e6e);
+  width: 11%;
+  border-top-left-radius: 5px;
+  border-bottom-left-radius: 5px;
+  border: 1px gray solid;
+  text-align: center;
+}
+
+.normal-box {
+  background: linear-gradient(to top, #6e6e6e, #4d4d4d, #6b6b6b 55%, #6e6e6e 45%, #707070, #6e6e6e);
+  width: 11%;
+  border: 1px gray solid;
+}
+
+.search-box {
+  background: linear-gradient(to top, #6e6e6e, #4d4d4d, #6b6b6b 55%, #6e6e6e 45%, #707070, #6e6e6e);
+  width: 12%;
+  align-items: center;
+  border-top-right-radius: 5px;
+  border-bottom-right-radius: 5px;
+  border: 1px gray solid;
+}
+
+.normal-box-1:hover,
+.normal-box:hover {
+  background: linear-gradient(to top, #4d4d4d, #2c2c2c, #4d4d4d 55%, #4d4d4d 45%, #525252, #4d4d4d);
+}
+
+.search-bar {
+  width: 70%;
+  background-color: gray;
+  border-radius: 25px;
+  padding-right: 12px;
+  margin-left: 10px;
+  margin-bottom: 2px;
+  text-align: center;
+  border: 1px #4d4d4d solid;
+}
+
+.search-box i {
+  position: absolute;
+  color: white;
+  padding-left: 17px;
+  font-size: 11px;
+}
+
+.verse {
+  text-align: center;
+}
+
+.verse a {
+  text-decoration: none;
+  color: white;
+  font-weight: 100;
+}
+
+.verse div {
+  margin-bottom: 50px;
+}
+
+.question {
+  font-size: 62px;
+  font-weight: 100;
+  margin: 10px;
+  margin-top: 100px;
+}
+
+.main-text {
+  font-size: 30px;
+  font-weight: 100;
+  line-height: 10px;
+}
+
+.video-text {
+  font-size: 23px;
+}
+
+.video-text:hover {
+  text-decoration: underline;
+}
+
+/* End of Main Section */
+
+/* Bottom Section */
+
+.bottom-section {
+  width: 80%;
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  width: 100%;
+}
+
+.grid img {
+  width: 100%;
+  height: auto;
+}
+
+/* End of Bottom Section */
+
+/* Footer Section */
+
+.footer-section {
+  width: 80%;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  font-size: 11px;
+}
+
+.right-box {
+  padding: 0;
+  margin-top: 20px;
+}
+
+.left-box {
+  justify-content: flex-end;
+  padding-right: 0;
+  margin-top: 20px;
+}
```

Comment:
- [ ] Your footer doesn't look centered, you can fix this by increasing the width in your left-box: 
![image](https://user-images.githubusercontent.com/53256139/89357456-27357d00-d686-11ea-9d3a-63cf57f4d432.png)